src/game/client/components/chat.cpp

```diff
@@ -576,7 +576,7 @@ bool CChat::LineShouldHighlight(const char *pLine, const char *pName)
 	return false;
 }
 
-const char *SAVES_HEADER[] = {
+static constexpr const char *SAVES_HEADER[] = {
```

Comment:
Also make the pointers const:

```suggestion
static constexpr const char *const SAVES_HEADER[] = {
```

Reply:
Wasn't sure if this should be the case

Reply:
Apparently that change was unnecessary. Looks like the `constexpr` already implies that the pointers will also be const.

```
static const char *SAVES_HEADER[] = { // compiles
// static const char *const SAVES_HEADER[] = { // doesn't compile
// static constexpr const char *SAVES_HEADER[] = { // doesn't compile
	"Time",
	"Player",
	"Map",
	"Code",
};

int main() {
    SAVES_HEADER[0] = "something else";
}
```

(https://godbolt.org/)

So I guess using the original would be better because it avoids the redundant `const`.